### ISSUE TITLE
Status of uninstallable revdeps

### ISSUE THREAD

**Issue by rstub** [r-lib/revdepcheck#219]
I recently ran `revdepcheck::revdep_check()` on `tikzDevice` and got the following output:

```
── CHECK ──────────────────────────────────────────────────────── 9 packages ──
✔ CPAT 0.1.0                             ── E: 0     | W: 0     | N: 1
✔ dplR 1.7.0                             ── E: 0     | W: 0     | N: 2
✔ dtwSat 0.2.5                           ── E: 1     | W: 0     | N: 0 
✔ gMOIP 1.3.0                            ── E: 1     | W: 0     | N: 0 
✔ knitr 1.23                             ── E: 0     | W: 0     | N: 2
✔ lineqGPR 0.0.4                         ── E: 0     | W: 0     | N: 0    
✔ mkin 0.9.49.5                          ── E: 0     | W: 0     | N: 0  
✔ RankingProject 0.1.1                   ── E: 0     | W: 0     | N: 0      
✔ wrswoR 1.1                             ── E: 0     | W: 0     | N: 0 
OK: 9
BROKEN: 0
```

as well as 

```r
> revdepcheck::revdep_report_cran()
## revdepcheck results

We checked 9 reverse dependencies, comparing R CMD check results across CRAN and dev versions of this package.

 * We saw 0 new problems
 * We failed to check 0 packages
```

Inspecting the two packages with an `E` (dtwSat and gMOIP) I found that these could not be installed due to some missing system library. My expectation would have been that this should be reported as "We failed to check 2 packages". When is a package reported as "failed to check"?

**Comment by gaborcsardi** [comment]
> When is a package reported as "failed to check"?

If it could not start `R CMD check`.

**Comment by rstub** [comment]
OK, the `R CMD check` runs for these packages were short, but it was possible to call them. And since the packages produced an error for both old and new version, it is indeed understandable to mark these checks as OK.

Anyway, it would be great if `revdepcheck` would identify the situation where a revdep could not be installed, but feel free to close this issue if this appears unreasonable to you.

**Comment by gaborcsardi** [comment]
I guess it would indeed make sense to mark these as "could not check", so I'll leave this open.

**Comment by MLopez-Ibanez** [comment]
Same as #256
